### DISCUSSION TITLE
Remap Keys in Interactive Mode

### DISCUSSION THREAD

**Discussion by Balrrach** [kovidgoyal/kitty#4803] [Q&A] [answered]
Hello. I use colemak and have motion keys remapped everywhere. I want to know if it is possible to remap keys in bash interactive mode from kitty. An example of bash interactive mode would be the mode in which one ends after executing a command like `man ls` in which keys have a different functions than inserting text.

**Comment by page-down** [comment, marked as answer]
> ... remap keys in bash interactive mode from kitty ...

You need to configure bash.
In kitty you can configure certain keys to send different keys.
Then you can set the function of the changed keys in bash.
This is global by default, but of course you can write kitten to only send specific keys when running bash.

> ... the mode in which one ends after executing a command like man ls in which keys have a different functions ...

After you run `man ls`, it has nothing to do with bash. These are the functions from man or some pager.

**Reply by Balrrach** [reply]
Thanks!
